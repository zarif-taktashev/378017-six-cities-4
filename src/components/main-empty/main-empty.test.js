import * as React from 'react';
import * as renderer from 'react-test-renderer';
import {Router} from 'react-router-dom';
import MainEmpty from './main-empty.jsx';
import history from '../../history';

it(`check render MainEmpty`, () => {
  const tree = renderer.create(
      <Router
        history={history}
      >
        <MainEmpty
          currentCity={`Amstedam`}
        />
      </Router>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
