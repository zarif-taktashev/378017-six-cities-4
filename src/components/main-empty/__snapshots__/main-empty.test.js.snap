// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check render MainEmpty 1`] = `
<div
  className="cities__places-container cities__places-container--empty container"
>
  <section
    className="cities__no-places"
  >
    <div
      className="cities__status-wrapper tabs__content"
    >
      <b
        className="cities__status"
      >
        No places to stay available
      </b>
      <p
        className="cities__status-description"
      >
        We could not find any property availbale at the moment in 
        Amstedam
      </p>
    </div>
  </section>
  <div
    className="cities__right-section"
    style={
      Object {
        "backgroundImage": "url(./img/no-places@2x.png)",
        "backgroundPosition": "right 100%",
        "backgroundRepeat": "no-repeat",
        "backgroundSize": "auto 119%",
      }
    }
  />
</div>
`;
